rsync_to_rdisc.py

```diff
@@ -334,24 +335,35 @@ def upload_exomedepth_vcf(run, run_folder):
 
 
 if __name__ == "__main__":
-
-    """If daemon is running exit, else create transfer.running file and continue."""
+    # If daemon is running exit, else create transfer.running file and continue.
     run_file = check_daemon_running(settings.wkdir)
+    remove_run_file = True
 
-    """Check if mount to BGarray intact."""
-    check_mount(settings.bgarray, run_file)
-
-    """Make set of transferred_runs.txt file, or create transferred_runs.txt if not present."""
+    # Make set of transferred_runs.txt file, or create transferred_runs.txt if not present.
     transferred_set = get_transferred_runs(settings.wkdir)
 
-    """Get folders to be transfer from HPC."""
+    # Connect to hpc
     client, hpc_server = connect_to_remote_server(settings.host_keys, settings.server, settings.user, run_file)
-    to_be_transferred = get_folders_remote_server(client, settings.folder_dic, run_file, transferred_set)
 
-    """Rsync folders from HPC to bgarray."""
-    remove_run_file = rsync_server_remote(hpc_server, client, to_be_transferred, run_file)
+    # Run rsync commands for each mount point.
+    for mount_name in settings.transfer_settings:
+        mount_path = settings.transfer_settings[mount_name]['mount_path']
+        # Check if mount is available and continue
+        if is_mount_available(mount_name, mount_path, run_file):
+            # Get folders to be transferred
+            to_be_transferred = get_folders_remote_server(
+                client,
+                settings.transfer_settings[mount_name],
+                run_file,
+                transferred_set
+            )
+
+            # Rsync folders from HPC to mount
+            rsync_succes = rsync_server_remote(hpc_server, client, to_be_transferred, mount_path, run_file)
+            if not rsync_succes:
+                remove_run_file = False
```

Comment:
Originally, a new rsync_to_rdisc process would be blocked when a mount was unavailable. Someone has to take a look and fix the mount, and I think to avoid many emails as well.
I think the new code changes this scenario; the sys.exit() is removed in `is_mount_available`. 
Can be resolved with:
```
else:
    remove_run_file = False
```

Reply:
I see. Let's think about a solution that continues bgarray transfer if glims is unavailable, and avoid multiple emails if a mount is lost. :)